boards/arm/mimxrt595_evk/mimxrt595_evk_cm33.dts

```diff
@@ -368,23 +362,23 @@ zephyr_udc0: &usbhs {
 
 			boot_partition: partition@0 {
 				label = "mcuboot";
-				reg = <0x00000000 DT_SIZE_K(64)>;
+				reg = <0x00000000 DT_SIZE_K(256)>;
 			};
-			slot0_partition: partition@10000 {
+			slot0_partition: partition@40000 {
 				label = "image-0";
-				reg = <0x00010000 DT_SIZE_M(24)>;
+				reg = <0x00040000 DT_SIZE_M(3)>;
 			};
-			slot1_partition: partition@1810000 {
+			slot1_partition: partition@340000 {
 				label = "image-1";
-				reg = <0x01810000 DT_SIZE_M(24)>;
+				reg = <0x00340000 DT_SIZE_M(3)>;
 			};
-			scratch_partition: partition@3010000 {
+			scratch_partition: partition@640000 {
 				label = "image-scratch";
-				reg = <0x03010000 DT_SIZE_K(8128)>;
+				reg = <0x00640000 DT_SIZE_K(768)>;
 			};
```

Comment:
Scratch can go unless the slots have different sector sizes, mcuboot should be using swap using move

Reply:
Scratch partition removed in latest push